blaze/expr/tests/test_collections.py

```diff
@@ -108,13 +108,21 @@ def test_join_option_types():
     assert join(b, a, 'x').dshape == dshape('var * {x: int}')
 
 
-def test_join_mismatched_schema():
+def test_join_exceptions():
+    'mismatched schema; no shared fields'
```

Comment:
that is fine, we might wasa to just check that the expected column names appear in the string